### ISSUE TITLE
Character lists not detecting deaths/dead characters

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#87]
```
jQuery.Deferred exception: Cannot read property 'indexOf' of undefined TypeError: Cannot read property 'indexOf' of undefined
    at Mule.parse (https://jakcodex.github.io/muledump/lib/mule.js:1008:72)
    at Mule.query (https://jakcodex.github.io/muledump/lib/mule.js:558:26)
    at Mule.queueStart (https://jakcodex.github.io/muledump/lib/mule.js:369:44)
    at Object.setuptools.init.accounts (https://jakcodex.github.io/muledump/lib/setuptools/src/init.js:225:31)
    at HTMLDocument.<anonymous> (https://jakcodex.github.io/muledump/lib/muledump.js:349:25)
    at j (https://jakcodex.github.io/muledump/vendor/jquery-3.2.1.min.js:2:29999)
    ...
```

**Comment by jakcodex** [comment]
There was no check for if the active list actually existed in the object.

**Comment by jakcodex** [comment]
Released in v9.0.0
